docs/src/main/sphinx/release/release-396.md

```diff
@@ -0,0 +1,51 @@
+# Release 396 (14 Sep 2022)
+
+## General
+
+* Improve performance of queries that process string data types. ({issue}`12798`)
+* Fix failure when querying a view which uses a table function.  ({issue}`13944`)
+
+## BigQuery connector
+
+* Add support for [column comments and table comments](/sql/comment). ({issue}`13882`)
+
+## ClickHouse connector
+
+* Allow pushdown of larger predicates by default. ({issue}`14029`)
+* Extend the range of possible values of the ClickHouse `Date` and `DateTime`
+  types. ({issue}`11116`)
+
+## Delta Lake connector
+
+* Improve optimized Parquet writer performance for
+  [structural data types](https://trino.io/docs/current/language/types.html#structural-data-types). ({issue}`13714`)
+* Fix performance regression in reading Parquet files. ({issue}`14094`)
+
+## Hive connector
+
+* Add verification for files written by the optimized Parquet writer. ({issue}`13246`)
+* Improve optimized Parquet writer performance for
+  [structural data types](https://trino.io/docs/current/language/types.html#structural-data-types). ({issue}`13714`)
+* Fix performance regression in reading Parquet files. ({issue}`14094`)
+
+## Iceberg connector
+
+* Improve performance when writing Parquet files with
```

Comment:
Same comment as with the Delta Lake connector --  I think there's no other writer for the Iceberg connector.